README.md

```diff
@@ -1,8 +1,32 @@
 # SymPy Paper
 
-[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+This is the source repository for the paper:
+
+> SymPy Authors, "SymPy: Symbolic Computing in Python", 2016.
+
+This repository contains or links to all of the information needed to reproduce
+the results in the paper.
+
+The latest rendered version of the PDF can be viewed via the
+[ShareLaTeX](http://sharelatex.com) CI system:
 
 [![PDF Status](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/badge.svg)](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/output.pdf)
 
+# Contributing
+
+Join our chat room to discuss contributions:
+
+[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# License
+
+The contents of this repository is licensed under the [Creative Commons
+Attribution 4.0 International
+License](http://creativecommons.org/licenses/by/4.0).
+
+[![CCBY](https://i.creativecommons.org/l/by/4.0/80x15.png)](http://creativecommons.org/licenses/by/4.0)
```

Comment:
I don't know what you mean but "is this license open source"?

We can use one of the typical code licenses like MIT or BSD, but CC is much more common for written works. For example, if we publish this open access, it will be licensed with a CC license, not a software license.

It may not really matter practically if we choose MIT, BSD, or CC-BY (as the all are essentially saying the same thing), but I've rarely seen software licenses applied to academic papers. That's why I have suggested CC-BY here.